wpilibc/shared/include/Commands/Command.h

```diff
@@ -66,8 +66,7 @@ class Command : public ErrorBase, public NamedSendable, public ITableListener {
   bool IsInterruptible() const;
   void SetInterruptible(bool interruptible);
   bool DoesRequire(Subsystem* subsystem) const;
-  typedef std::set<Subsystem*> SubsystemSet;
-  SubsystemSet GetRequirements() const;
```

Comment:
Why remove the typedef?  This is good practice so we can change the type without breaking the API.